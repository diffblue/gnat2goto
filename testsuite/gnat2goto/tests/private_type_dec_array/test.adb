with Private_Dec_Array; use Private_Dec_Array;

procedure Test is
   MyNum : Integer := 5;
begin
   P (MyNum);
end Test;
