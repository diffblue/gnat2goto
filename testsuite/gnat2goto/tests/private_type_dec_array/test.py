from test_support import *

prove(main="test.adb")
