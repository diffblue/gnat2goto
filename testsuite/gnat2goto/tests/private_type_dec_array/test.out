[p.assertion.1] line 5 Ada Check assertion: SUCCESS
VERIFICATION SUCCESSFUL
